npm/iptm/iptm.go

```diff
@@ -298,6 +299,8 @@ func (iptMgr *IptablesManager) DeleteChain(chain string) error {
 
 // Add adds a rule in iptables.
 func (iptMgr *IptablesManager) Add(entry *IptEntry) error {
+	timer := metrics.StartNewTimer()
```

Comment:
Can you also run your change against the load test. Please sync with Shufang for how to run the scale tests?